manager/install_cortex.sh

```diff
@@ -153,6 +153,7 @@ function validate_cortex() {
         continue
       fi
       operator_endpoint_reachable="ready"
+      echo "{\"cortex_url\": \"$operator_endpoint\", \"aws_access_key_id\": \"$CORTEX_AWS_ACCESS_KEY_ID\", \"aws_secret_access_key\": \"$CORTEX_AWS_SECRET_ACCESS_KEY\"}" > /.cortex/dev.json
```

Comment:
I think we should alert the user that we configured their CLI, e.g.

```bash
echo -e "\n✓ Configured CLI"
```

Feel free to run it by Omer